src/styles/getModalStyles.js

```diff
@@ -96,6 +96,24 @@ export default (type, windowDimensions) => {
             animationIn = 'fadeInLeft';
             animationOut = 'fadeOutLeft';
             break;
+        case CONST.MODAL.MODAL_TYPE.RIGHT_DOCKED:
+            modalStyle = {
+                margin: 0,
+                flexDirection: 'row-reverse',
+                justifyContent: 'end',
+            };
+            modalContainerStyle = {
+                width: isSmallScreen ? '100%' : '40%',
+                height: '100%',
+                overflow: 'hidden',
+                marginRight: 0,
+                paddingTop: 0,
```

Comment:
NAB, I haven't looked yet but who/what is adding margin and padding that we need to reset it? Maybe it would be better to only add margin/padding to the places that need it instead of override it here.